routes/api/v1.php

```diff
@@ -122,6 +123,7 @@
         'users/{user_id}/preferences' => 'User\PreferenceController',
         'users/{user_id}/assos' => 'User\AssoController',
         'users/{user_id}/services' => 'User\ServiceController',
+        'users/{user_id}/contributions' => 'User\ContributionController',
```

Comment:
If you are generating all routes for your resource, you need also to create others methods: `store, update, delete`

Check `VisibilityController` ;)